src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs

```diff
@@ -47,4 +47,23 @@ public class InputFilteredEventArgs : EventArgs
         [EditorBrowsable(EditorBrowsableState.Never)]
         public InputFilterType Type { get; set; }
     }
+
+    /// <summary>
+    /// The CursorPositionChanged event arguments.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class CursorPositionChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// cursor postion before the change.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint OldCursorPosition { get; set; }
```

Comment:
I can't found any event args that provide old value in NUI. for the consistency how about remove OldCursorPosition?